src/tests/ftest/pool/destroy_rebuild.py

```diff
@@ -48,8 +48,7 @@ def test_destroy_while_rebuilding(self):
         :avocado: tags=all,pr,medium,pool,destroypoolrebuild
         """
         # Get the test parameters
-        self.pool = TestPool(self.context, self.log,
-                             dmg_command=self.get_dmg_command())
+        self.pool = TestPool(self.context, self.get_dmg_command(), log=self.log)
```

Comment:
The `log` argument is defunct for TestPool.create():
```suggestion
        self.pool = TestPool(self.context, self.get_dmg_command())
```